strongbox-aql/pom.xml

```diff
@@ -40,22 +40,108 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>aql-dynamic-grammar</id>
+                        <goals>
+                            <goal>execute</goal>
+                        </goals>
+                        <phase>generate-sources</phase>
+                    </execution>
+                </executions>
+                <configuration>
+                    <scripts>
+                        <script>file:///${basedir}/src/main/groovy/aql-dynamic-grammar.groovy</script>
+                    </scripts>
+                </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>${project.groupId}</groupId>
+                        <artifactId>strongbox-storage-maven-layout-provider</artifactId>
+                        <version>${project.version}</version>
```

Comment:
I think you can remove the versions from these dependencies.

Reply:
unfortunately it's not working without versions
```
[ERROR]   The project org.carlspring.strongbox:strongbox-aql:1.0-SNAPSHOT (/home/sbespalov/projects/strongbox/strogbox/strongbox-aql/pom.xml) has 5 errors
[ERROR]     'build.plugins.plugin[org.codehaus.gmavenplus:gmavenplus-plugin].dependencies.dependency.version' for org.carlspring.strongbox:strongbox-storage-maven-layout-provider:jar is missing. @ org.carlspring.strongbox:strongbox-aql:[unknown-version], /home/sbespalov/projects/strongbox/strogbox/strongbox-aql/pom.xml, line 61, column 33
```